src/Illuminate/Container/Container.php

```diff
@@ -285,11 +286,17 @@ public function bindMethod($method, $callback)
      *
      * @param  array|string $method
      * @return string
+     *
+     * @throws \InvalidArgumentException
      */
     protected function parseBindMethod($method)
     {
         if (is_array($method)) {
-            return $method[0].'@'.$method[1];
+            if (($count = count($method)) > 1) {
```

Comment:
I think its better to throw exception in conditional, and place return after it.

```php
if (is_array($method)) {
    if (($count = count($method)) < 2) {
        throw new InvalidArgumentException("Method should be array with length >= 2, array with length [$count] given");
    }

    return $method[0].'@'.$method[1];
}
```

Reply:
@a-komarev , changed